scripts/cnode-helper-scripts/cntools.sh

```diff
@@ -1255,10 +1223,64 @@ case $OPERATION in
       say "${RED}ERROR${NC}: invalid URL format or more than 64 chars in length"
       waitForInput && continue
     fi
-    
-    say "\n${ORANGE}Please make sure you host your metadata JSON file (with contents as below) at ${meta_json_url} :${NC}\n"
-    say "{\n  \"name\": \"${meta_name}\",\n  \"ticker\": \"${meta_ticker}\",\n  \"description\": \"${meta_description}\",\n  \"homepage\": \"${meta_homepage}\"\n}" | tee "${pool_meta_file}"
-    
+
+    metadata_done=false
+    if wget -q  "$meta_json_url" -P "$pool_meta_save_location"; then
+      say "\nMetadata exists at URL.  Use existing data?\n"
+      case $(select_opt "[y] Yes" "[n] No") in
+        0) filename=$(basename "$meta_json_url")
+           pool_meta_file=${pool_meta_save_location}${filename}
+           metadata_done=true
+           ;;
+        1) : ;; # Do nothing
+      esac
+
+    fi
+    if [ ${metadata_done} = false ]; then
+
+      if [[ -f "${pool_meta_file}" ]]; then
+        meta_name=$(jq -r .name "${pool_meta_file}")
+        meta_ticker=$(jq -r .ticker "${pool_meta_file}")
+        meta_homepage=$(jq -r .homepage "${pool_meta_file}")
+        meta_description=$(jq -r .description "${pool_meta_file}")
+      fi
```

Comment:
During this validation, as per tests from @hodlonaut  - while testing large dummy JSON queries, we found that not all fields were extracted by cardano-wallet (thus, I assume SMASH too). Perhaps a #TODO comment to align with wallet in future - in case of discrepancy

Reply:
I'm not clear what you want changed here.  just add a `#TODO` comment around this section of code?

Reply:
added a simple comment